alf/algorithms/sac_algorithm.py

```diff
@@ -425,7 +426,7 @@ def _calc_critic_loss(self, training_info):
             value=critic_info.critic2,
             target_value=target_critic)
 
-        critic_loss = critic_loss1.loss + critic_loss2.loss
+        critic_loss = (critic_loss1.loss + critic_loss2.loss) / 2.
```

Comment:
``` / 2``` should be removed as each critic network is trained using its original loss instead of a scaled loss.

Reply:
The absolute scale is not important. We can also argument MSELoss * 0.5 is the way to go. More importantly, the current displayed critic_loss in TB is always doubled, which is confusing.

Reply:
Changed to only scaling the extra field